benchmark/wildtime_benchmarks/data_generation_yearbook.py

```diff
@@ -85,9 +122,9 @@ def create_binary_file(data, output_file_name: str, timestamp: int) -> None:
                 label_integer = tensor2.item()
 
                 features_size = len(features_bytes)
-                assert features_size == 4096
+                assert features_size == 12288
 
-                f.write(int.to_bytes(label_integer, length=4, byteorder="big"))
+                f.write(int.to_bytes(label_integer, length=4, byteorder="little"))
```

Comment:
I think after merging #398 we should default to big since it's cheaper to parse the big endian files at the storage (no reverse iterator needed)